### ISSUE TITLE
[enterprise-4.12] Issue in file networking/network_observability/network-observability-overview.adoc

### ISSUE THREAD

**Issue by eranra** [openshift/openshift-docs#54811]
### Which section(s) is the issue in?

Updated text for "Network Observability metrics" 

@skrthomas @stleerh 

### What needs fixing?

```
Network Observability metrics

The OpenShift Container Platform console offers an Overview tab that displays various metrics for the network traffic in the cluster. The information can be displayed by node, namespace, owner, pod, and service. In addition, it is possible to refine the displayed metrics using filters and display options.

```

**Comment by lmurthyrh** [comment]
@skrthomas  - For your attention

**Comment by skrthomas** [comment]
Closing this issue, as it is addressed with #55076
